sqlmesh/core/renderer.py

```diff
@@ -69,6 +71,7 @@ def __init__(
         self._only_execution_time = only_execution_time
 
         self._cache: t.Dict[t.Tuple[datetime, datetime, datetime], t.List[exp.Expression]] = {}
+        self._models: t.Optional[UniqueKeyDict[str, Model]] = None
```

Comment:
I'm somewhat confused: who sets this when we render a model from a snapshot? I can only see this being set once during the load.